demo/hacktoberfest-custom-themes/js/theme.js

```diff
@@ -253,7 +253,6 @@ themes.set("2022", new Map([
             stroke: colors2022.secondaryPsybeam,
         },
         exclusiveGateway: {
-            fill: colors2022.secondaryManga,
```

Comment:
**question:** why removing this? here we are supposed to only change technical things, not configuration

Reply:
Because the property `secondaryManga` doesn't exist 😆

Reply:
Fair enough 😜